Wireframe/index.html

```diff
@@ -4,10 +4,35 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <main>
+      <section class="toptitle">
+        <h1 class="heading">Understanding GIT</h1>
+        <p class="heading_summary">This page will tell you the basics of GIT and why it is useful.</p>
+      </section>
+      <section class="HERO">
+        <section class="hero">
+          <h2 class="article_title">What is GIT?</h2>
+          <p class="article_summary">GIT is a distributed version control system. What that means is that it is a software used to track changes in any given file. It allows programmers to collaborate and work together at the same time and gives them a chance to monitor others work.</p>
+          <a href="https://www.w3schools.com/git/git_intro.asp?remote=github"><button class="article_button">Read more</button></a>
+        </section>
+        <section class="sub_hero">
+          <section class="category1"
+            <h2 class="article_title">Why do developers need GIT?</h2>
+            <p class="article_summary">Developers use GIT for several reasons. One of the main reason is it allows them to keep track of the changes made and revert back to the previous version if needs be.</p>
+            <a href="https://www.nobledesktop.com/learn/git/what-is-git#:~:text=Git%20allows%20users%20to%20'stage,smoother%20collaboration%20with%20other%20developers."><button class="article_button">Read more</button></a>
+          </section>
+          <section class="category2"
+            <h2 class="article_title">What is branch in GIT?</h2>
+            <p class="article_summary">A branch is a new seperate version of the main repository. It allows you to make any changes needed and then merge your code onto the main brainch for reviewing.</p>
+            <a href="https://www.w3schools.com/git/git_branch.asp?remote=github"><button class="article_button">Read more</button></a>
+          </section>
+        </section>
+      </section>
+    </main>
+    <footer>
```

Comment:
Why is this footer empty? Should we include empty elements in html? What happens if we do this?

Reply:
I have sorted this out. I wanted to include a footer because all pages should have header main and footer but I did not know what to include in this, II have sorted this out now.